### PR TITLE
chore(plugins): restore the base folke/lua-dev

### PR DESCRIPTION
**Pull request by kylo252**
# Description

Restore the original base folke/lua-dev.nvim

--- 

Note: an interesting commit https://github.com/hrsh7th/nvim-cmp/commit/8fc8d31cf451b8becb39087aa2190bee29a8b009

### CODE REVIEW

**Comment by kylo252** [comment]
@abzcoding, we can probably wait with this until tomorrow in case there are more updates.

**Review by abzcoding — APPROVED** [review]
I agree. We probably should wait a bit.

File: lua/lvim/plugins.lua [hunk 1]
```diff
@@ -10,19 +10,19 @@ local commit = {
...
-  nvim_cmp = "4c0a6512a0f8a235213959badf70031b9fa0220a",
+  nvim_cmp = "8fc8d31cf451b8becb39087aa2190bee29a8b009",
   nvim_dap = "08349ab5da2148965deb2e778065017059bff584",
-  nvim_lsp_installer = "9894db12783fdd02be909cdd60e6d0fc20b9e46e",
+  nvim_lsp_installer = "524553022ca5ad29608dab9568012076aa17ee0d",
   nvim_lspconfig = "09130804eb27fd31539922967d14c28388765ae4",
   nvim_notify = "15f52efacd169ea26b0f4070451d3ea53f98cd5a",
   nvim_tree = "0a2f6b0b6ba558a88c77a6b262af647760e6eca8",
-  nvim_treesitter = "723d91e8217ae66ea75f809f404d801ed939f497",
+  nvim_treesitter = "8b1d3906199e1572c2201e494bba017d823c0c8f",
```
Comment (abzcoding) [inline lua/lvim/plugins.lua:25]: btw this causes weird issues for neovim head. Can we keep nvim_treesitter for another PR to make sure it doesn't break anything

**Review by abzcoding — CHANGES_REQUESTED** [review]

**Comment by abzcoding** [comment]
@kylo252 can we keep treesitter before https://github.com/nvim-treesitter/nvim-treesitter/commit/c80715f883b8c7963782973b23297c5dec7924be

it causes all sorts of errors atm in neovim head

**Comment by abzcoding** [comment]
Screen Shot 2022-01-19 at 1 35 17 AM

**Comment by abzcoding** [comment]
we can probably run the `make generate_plugins_sha` one more time, before merging this, to make sure we are adding all the changes :)

**Comment by kylo252** [comment]
> we can probably run the `make generate_plugins_sha` one more time, before merging this, to make sure we are adding all the changes :)

Another treesitter commit 👀

**Comment by abzcoding** [comment]
LGTM 🚀

**Comment by abzcoding** [comment]
we should somehow remind users to do a `:TSUpdate` as well 🤔

**Comment by kylo252** [comment]
> we should somehow remind users to do a `:TSUpdate` as well thinking

I was going to write the same thing, but I forgot that It's not needed if you're running `:LvimUpdate` :sweat_smile: 

https://github.com/LunarVim/LunarVim/blob/56df185b1b9ec9c620e80654d314541beca76f26/lua/lvim/utils/hooks.lua#L45-L61

**Pull request merged**